docker-compose.yml

```diff
@@ -0,0 +1,5 @@
+tests:
```

Comment:
Do we really need docker compose for that? This add another dependency for new developers. This should be simple enough to just have a rake task for that. Feel free to modify the task I added to master recently.